src/tests/test_tls_stream_integration.cpp

```diff
@@ -203,7 +256,7 @@ class Server : public Side, public std::enable_shared_from_this<Server>
          // Note: If this was a real server, we should create a new session (with its own stream) for each accepted
          // connection. In this test we only have one connection.
 
-         if (m_move_before_accept)
+         if(m_move_before_accept)
```

Comment:
removal of space probably not intended.

Reply:
Coding style doesn't have a space there.